charts/steward/templates/podmonitor-jenkinsfile-runner.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.metrics.podMonitors.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: log-forwarder
```

Comment:
Naming is a bit off between the resource name and the file name.

Reply:
Renamed here https://github.com/SAP/stewardci-core/pull/410/commits/406df9317653d079d855b78241a5d1a1e80a2327